profile_test.go

```diff
@@ -0,0 +1,281 @@
+package eat
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+const (
+	InvalidProfile = 100
+	InvalidURL     = "abcd"
+	EmptyURL       = "  "
+	InvalidOID     = "xxxx"
+	EmptyOID       = "  "
+)
+
+// TestProfileURL_Basic tests the basic setting of Profile as URL string
+func TestProfileURL_Basic(t *testing.T) {
+	assert := assert.New(t)
```

Comment:
this shadows the `assert` symbol from the imported package.  You can pass `t` as the first parameter to each `assert` method instead
```suggestion
```

Reply:
Done!